### PR TITLE
Update set-password-expiration-policy.md

### PR DESCRIPTION
**Pull request by being-a-geek**
Security and Privacy tab has moved under Settings> Org Settings in the Microsoft 365 Admin Portal

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ If you're a user, you don't have the permissions to set your password to never e
 
 Follow the steps below if you want to set user passwords to expire after a specific amount of time.
 
-1. In the Microsoft 365 admin center, go to the <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">**Security & privacy** tab</a>.
+1. In the Microsoft 365 admin center, go to Settings, Org Settings, and then the <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">**Security & privacy** tab</a>.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
1. In the Microsoft 365 admin center, go to **Settings** > **Org Settings**, and then select the <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">**Security & privacy** tab</a>.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @being-a-geek, kindly commit the suggested change. Many thanks.

**Comment by being-a-geek** [comment]
> Copy review - @being-a-geek, kindly commit the suggested change. Many thanks.
@JohanFreelancer9 
committed the suggested change

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**